src/Integration/Binding/BindingController.cs

```diff
@@ -157,9 +178,9 @@ private void OnBindingFinished(BindCommandArgs bindingArgs, bool isFinishedSucce
             {
                 this.host.VisualStateManager.SetBoundProject(bindingArgs.Connection.ServerUri, bindingArgs.Connection.Organization?.Key, bindingArgs.ProjectKey);
 
-                // TODO: CM2: the conflicts controller is only applicable in legacy connected mode
-                // However, it *should* be safe to call it regardless - in new connected mode it should
-                // return false.
+                // The conflicts controller is only applicable in legacy connected mode
```

Comment:
I manually tested this code path: the conflict controller checks whether it is in legacy connected mode and returns an empty list if it isn't. I decided not to repeat the same check here.